src/OrchardCore.Modules/OrchardCore.Queries/Views/Query.Fields.Edit.cshtml

```diff
@@ -10,6 +10,6 @@
 
 <fieldset class="form-group">
     <label asp-for="Schema">@T["Schema"]</label>
-    <textarea asp-for="Schema" rows="12" class="form-control"></textarea>
+    <textarea asp-for="Schema" rows="12" class="form-control" style="text-align:left"></textarea>
```

Comment:
Is that necessary since you changed it in the codemirror css?

Reply:
Yep, this is for Schema textarea I made it inline style to enforce this field to align to left even in RTL

Reply:
ok